### PR TITLE
general nitpick-y changes, nginx conf and sticky header

### LINKED ISSUE #12: nginx url rewrites

### PR DESCRIPTION
**Pull request by bernkastel02**
i want to format the css since its inconsistently spaced but ill do tht later
feel free to make reviews

### CODE REVIEW

File: static/css/pages/docs.css [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+    background-color:hsl(var(--main),10%,15%);
+    color:#fff;
+}
+
+.title-block {
+    height: auto;
+    width: auto;
+
+    padding: 30px 10px 10px 10px;
```
Comment (varkaria) [inline static/css/pages/docs.css]: i feel weird about padding
i think padding 22px is better

File: static/css/pages/docs.css [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    background-color: hsl(var(--main),50%,50%);
+}
+
+.doc-title {
+    font-size: 1.7em;
+    font-weight: 500;
+}
+
+.doc-block {
+    padding: 10px;
```
Comment (varkaria) [inline static/css/pages/docs.css]: this thing too
i think padding 22px is better

File: ext/nginx.example.conf [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+server {
+    listen 80;
+    #listen 443 ssl; # Include this if you want SSL support!
+
+    # The domain or URL you want this to run of f of.
+    server_name web.example.com;
+
+    # NOTE: You'll want to change these to your own SSL certificate if any.
+    #ssl_certificate     /srv/gulag-web/cert.cer;
+	#ssl_certificate_key /srv/gulag-web/key.pem;
```
Comment (varkaria) [inline ext/nginx.example.conf:13]: oh dat space

File: blueprints/frontend.py [hunk 4]
```diff
@@ -123,3 +123,13 @@ async def register_post():
...
+
+""" rules """
+@frontend.route('/rules') # GET
+async def rules():
+    return await render_template('rules.html')
+
+""" discord redirect """
+@frontend.route('/discord') # GET
+async def discord():
+    return redirect("https://discord.com/invite/Y5uPvcNpD9")
```
Comment (varkaria) [inline blueprints/frontend.py]: i will make some discord link config soon

**Review by varkaria — COMMENTED** [review]
### ok i commented
btw thank you so much to help our project
i think that's stuff i planned before 
but i didn't do it because i stuck in other stuff

File: static/css/pages/docs.css (hunk 1 above)
Comment (varkaria) [reply static/css/pages/docs.css]: i mean `padding: 30px 22px 10px 22px;`

File: blueprints/frontend.py (hunk 4 above)
Comment (bernkastel02) [reply blueprints/frontend.py]: its better to leave it there for now; you can easily change it whenever it's done (it's just there because 404 bad)

**Comment by varkaria** [comment]
ok lgtm 👍🏻

**Pull request merged**